### PR TITLE
Add Web3Auth Backend Docs

### PR DESCRIPTION
**Pull request by shahbaz17**

### CODE REVIEW

File: docs/sdk/web-backend.mdx [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+import Web3AuthOptions from "@site/src/common/sdk/web/_web3authcore-options.mdx";
+import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs";
+
+---
+
+## [`@web3auth/node-sdk`](https://npmjs.com/package/@web3auth/node-sdk)
+
+Web3Auth's `@web3auth/node-sdk` backend SDK is a simple and easy to use SDK to be used in your Node.js backend to get the same experience of Web3Auth
+frontend SDKs. One thing to note here is that, with this you get semi-custodiality.
```
Comment (yashovardhan) [inline docs/sdk/web-backend.mdx:17]: explain why this is semi custodial, maybe an architecture diagram will be a good addition. you can copy the one key flow diagram to edit the package name here. People need to know properly how this works.

**Review by yashovardhan — CHANGES_REQUESTED** [review]
Apart from the comment I made, I feel this doc is incomplete due to the following reasons:
- This doesn't follow our structure of other SDK references. 
- We need additional page around custom authentication - I feel this doesn't support standard web3auth verifiers, only custom authentication is supported - please check about this.
- separate installation, initialisation, custom authentication and usage as 4 different pages
- we need to also add in the usage page why multi factor authentication doesn't work here, separate out

**Comment by shahbaz17** [comment]
@yashovardhan 

Restructured the code and added more explanation on why this is needed.

**Review by yashovardhan — APPROVED** [review]
LGTM

**Pull request merged**